test/test_torchinductor_opinfo.py

```diff
@@ -40,7 +40,7 @@
 TestExpect = Enum("TestExpect", ("SUCCESS", "XFAILURE", "SKIP"))
 
 COLLECT_EXPECT = os.getenv("PYTORCH_COLLECT_EXPECT", "0") == "1"
-FAIL_ON_SUCCESS = os.getenv("PYTORCH_FAIL_ON_SUCCESS", "0") == "1"
+FAIL_ON_SUCCESS = os.getenv("PYTORCH_FAIL_ON_SUCCESS", "1") == "1"
```

Comment:
I am turning on FAIL_ON_SUCCESS...
If tests are flaky, we can revert

Reply:
The new CI results seem to suggest `__rdiv__` is flaky? You can move it to the skip list first.

Reply:
yes, done.